tests/integration/test_join.py

```diff
@@ -110,6 +110,22 @@ def test_join_right(c):
     assert_eq(return_df, expected_df, check_index=False)
 
 
+def test_join_cross(c, request, user_table_1, department_table):
```

Comment:
Looks like we can remove `request` as it isn't being used anymore:

```suggestion
def test_join_cross(c, user_table_1, department_table):
```